### PR TITLE
karmada operator: fixed karmada-aggregated-apiserver name

### PR DESCRIPTION
**Pull request by calvin0327**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

* typo: aggregrated -> aggregated

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: operator/pkg/tasks/init/apiserver.go [hunk 1]
```diff
@@ -89,31 +89,31 @@ func runWaitKarmadaAPIServer(r workflow.RunData) error {
...
 
-	err := apiserver.EnsureKarmadaAggregratedAPIServer(data.RemoteClient(), cfg, data.GetName(), data.GetNamespace())
+	err := apiserver.EnsureKarmadaAggregatedAPIServer(data.RemoteClient(), cfg, data.GetName(), data.GetNamespace())
 	if err != nil {
-		return fmt.Errorf("failed to install karmada aggregrated apiserver, err: %w", err)
+		return fmt.Errorf("failed to install karmada aggregated apiserver, err: %w", err)
 	}
 
-	klog.V(2).InfoS("[KarmadaAggregratedApiserve] Successfully installed karmada-aggregrated-apiserver component", "karmada", klog.KObj(data))
+	klog.V(2).InfoS("[KarmadaAggregatedApiserve] Successfully installed karmada-aggregated-apiserver component", "karmada", klog.KObj(data))
```
[the remark this answers is not in the document]
Comment (calvin0327) [reply operator/pkg/tasks/init/apiserver.go:109]: good found.
done

File: operator/pkg/constants/constants.go [hunk 2]
```diff
@@ -78,8 +78,8 @@ const (
 
 	// KarmadaAPIserverComponent defines the name of karmada apiserver component
 	KarmadaAPIserverComponent = "KarmadaAPIServer"
-	// KarmadaAggregratedAPIServerComponent defines the name of karmada aggregrated apiserver component
-	KarmadaAggregratedAPIServerComponent = "KarmadaAggregratedAAPIServer"
+	// KarmadaAggregatedAPIServerComponent defines the name of karmada aggregated apiserver component
+	KarmadaAggregatedAPIServerComponent = "KarmadaAggregatedAAPIServer"
```
Comment (Poor12) [inline operator/pkg/constants/constants.go:82]: KarmadaAggregatedAAPIServer => KarmadaAggregatedAPIServer

File: operator/pkg/tasks/deinit/component.go [hunk 3]
```diff
@@ -23,7 +23,7 @@ func NewRemoveComponentTask() workflow.Task {
 			newRemoveComponentSubTask(constants.KarmadaControllerManagerComponent, util.KarmadaControllerManagerName),
 			newRemoveComponentSubTask(constants.KubeControllerManagerComponent, util.KubeControllerManagerName),
 			newRemoveComponentWithServiceSubTask(constants.KarmadaWebhookComponent, util.KarmadaWebhookName),
-			newRemoveComponentWithServiceSubTask(constants.KarmadaAggregratedAPIServerComponent, util.KarmadaAggratedAPIServerName),
+			newRemoveComponentWithServiceSubTask(constants.KarmadaAggregatedAPIServerComponent, util.KarmadaAggratedAPIServerName),
```
Comment (Poor12) [inline operator/pkg/tasks/deinit/component.go:26]: KarmadaAggratedAPIServerName => KarmadaAggregatedAPIServerName

**Review by Poor12 — COMMENTED** [review]
Another two typos.

**Pull request merged**